README.md

```diff
@@ -352,6 +352,40 @@ DateFromString.decode('foo')
 
 Note that you can **deserialize** while validating.
 
+# Generic Types
+Polymorphic runtime types are represented using functions.
+For example, the following typescript:
+```ts
+interface ResponseBody<T> {
+  result: T[];
+  _links: Links;
+}
+interface Links {
+  previous: string;
+  next: string;
+}
+```
+Would be:
+```ts
+import * as t from 'io-ts';
+
+const ResponseBody = <RT extends iots.Mixed>(type: RT) =>
+  iots.interface({
+    result: iots.array(type),
```

Comment:
Should probably pass in the array like `ResponseBody(t.array(UserModel))` instead of assuming it's an array here.

Reply:
If you are representing

```ts
interface ResponseBody<T> {
  result: T[];
  _links: Links;
}
```

I think that the previous version was correct. The current implementation

```ts
const ResponseBody = <RT extends t.Mixed>(type: RT) =>
  t.interface({
    result: type,
    _links: Links
  });
```

is representing

```ts
interface ResponseBody<T> {
  result: T;
  _links: Links;
}
```

instead

Reply:
yeah I think it's better to use
```
interface ResponseBody<T> {
  result: T;
  _links: Links;
}
```
but I forgot to update the original interface, will do that!